newIDE/app/src/EventsBasedBehaviorEditor/EventsBasedBehaviorPropertiesEditor.js

```diff
@@ -729,6 +729,11 @@ export default function EventsBasedBehaviorPropertiesEditor({
                                           value="Boolean"
                                           label={t`Boolean (checkbox)`}
                                         />
+                                        <SelectOption
+                                          key="property-type-animationname"
+                                          value="AnimationName"
+                                          label={t`Animation Name (text)`}
+                                        />
```

Comment:
I wonder if it should be below the "Choice" type as it's a bit more specific.
```suggestion
                                        <SelectOption
                                          key="property-type-animationname"
                                          value="AnimationName"
                                          label={t`Animation name (text)`}
                                        />
```